delivery_tnt_oca/models/delivery_carrier.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import base64
+
+from odoo import _, fields, models
+
+from .tnt_request import TntRequest
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = "delivery.carrier"
+
+    delivery_type = fields.Selection(selection_add=[("tnt_oca", "TNT")])
+    tnt_oca_ws_username = fields.Char(string="ExpressConnect Username WS")
+    tnt_oca_ws_password = fields.Char(string="ExpressConnect Password WS")
+    tnt_oca_ws_account = fields.Char(string="ExpressConnect Account WS")
```

Comment:
What is this "WS"?

Reply:
WS = Webservice.
The names of the fields are similar to those used in other addons of similar carriers: `delivery_ups_oca` or `delivery_seur` for example.

Reply:
OK, then you need to reorder the name for it to make sense in English:

```suggestion
    tnt_oca_ws_username = fields.Char(string="ExpressConnect WS Username")
    tnt_oca_ws_password = fields.Char(string="ExpressConnect WS Password")
    tnt_oca_ws_account = fields.Char(string="ExpressConnect WS Account")
```

Reply:
Remove WS at all, as ExpressConnect is the webservice, and there can't be any other thing with that name.